### ISSUE TITLE
Markup

### ISSUE THREAD

**Issue by alnopa9** [lmcneil7/teen-titans#10]
## Stage Directions
In your markup, you left the parenthesis from the stage directions wrapping your text. Since you have actual `<stagDir>` element tags, the parenthesis are no longer needed.

## Speeches
You have the characters marked up, but not their speeches. I would include `<sp>` element tags for speeches.

## Semicolon
After the `<char>` element is marked, you still have the semicolon following the `</char>` tag. This is also not needed.

## `<char>` versus `<spkr>`
You want to differentiate between a character that is speaking and a character that is being referenced.
Line 78 on episode one,
```
<char>Robin</char> Want to give this guy the Sonic Boom? 
<stagDir>(<char>Cyborg</char> smiles and converts his arm to sonic cannon mode.)</stagDir>
```
Instead:
```
<sp><spkr><char>Robin</char></spkr>Want to give this guy the Sonic Boom?</sp>
<stagDir><char>Cyborg</char> smiles and converts his arm to sonic cannon mode.</stagDir>
```
Better yet:
```
<sp><spkr ref="#robin">Robin</spkr>Want to give this guy the Sonic Boom?</sp>
<stagDir><char>Cyborg</char> smiles and converts his arm to sonic cannon mode.</stagDir>
```

## `<act>`
For your act tag, it should wrap around the **entire** act, not just the title.
```
<act n="0">Prologue</act>
```
Instead:
```
<act n="0">
    <title>Prologue</title>
<!-- ~entire act~ -->
</act>
```

### New Markup
I marked up episode one with these changes in mind.
[XML for Episode One](https://github.com/lmcneil7/teen-titans/blob/master/example/episode-1.xml)
[Schema](https://github.com/lmcneil7/teen-titans/blob/master/example/schema.rnc)
[Step File](https://github.com/lmcneil7/teen-titans/blob/master/example/step-file.txt)
In the markup, I also included IDs and refs. The IDs for each character are in the `<characterList>` element for that character, and the `@ref` is located in the `<spkr>` tags.

**Comment by ebeshero** [comment]
@alnopa9 This new structure looks helpful so you can always be clear about who is speaking, and differentiate them from people being mentioned in a speech! It'll make XPath for the project a lot easier. @lmcneil7 and @smdunn921 what do you think?

**Comment by ebeshero** [comment]
Also, @lmcneil7 and @smdunn921, we hope this is okay to update the markup a little just now. What Alyssa is doing is basically we've been doing with a number of project teams this week--taking a look and seeing if we can make the markup a little easier to process and pull information with XPath!

**Comment by ebeshero** [comment]
Note: @alnopa9 set her schema inside the XML directory--it's sort of co-existing with your project schema at the moment. You'll probably want to rearrange the repo as you're reviewing the changes (move the schema where you want it, reapply schema associations, etc.)

**Comment by smdunn921** [comment]
Thank you for this! I’m having to look at it on my phone right now because my laptop is having wifi connection trouble, but when that’s fixed I’ll get all up in there.

**Comment by alnopa9** [comment]
Updated the links to reflect where I moved the example directory to.

**Comment by lmcneil7** [comment]
Sorry for the last response, but this is really helpful! We had something similar where we tried to add ref to certain <char> tags when it wasn't specified on which person it was talking about, but we couldn't figure how to make it work with the schema. Also, we decided it might make it easier to do if we didn't do it at all. However, your suggestion of adding another tag for characters (<spkr>) and use ref on that would help a lot. This might make markup even easier now so thank you for the suggestions!
